johnsonandjohnson/Bodiless-JS
# fix(test-site): Add Page padding on example site

## Changes
Page Padding was added to container

## Test Instructions
Static : Verify all pages have consistent padding
Edit Mode & Preview mode in edit: Verify all pages have consistent padding
 
## Related Issues
